lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1275,22 +1275,37 @@ def get_recommended_datastore(self, datastore_cluster_obj=None):
         Returns: Name of recommended datastore from the given datastore cluster
 
         """
+        datastore = None
```

Comment:
suggestion: move this line to https://github.com/ansible/ansible/pull/35644/files?diff=split#diff-ed147163551d1eaa084b21f6bab29f96R1299

Reply:
Done.